requirements/requirements.txt

```diff
@@ -66,32 +66,32 @@ stripe==1.12.2
 django-compressor==2.1
 django-angular==0.7.16
 Pycco==0.3.0
-django-mptt==0.8.6
+django-mptt==0.8.7
 jsonpath-rw==1.4.0
 pip>=6.1.1
 git+git://github.com/smartfile/django-transfer.git@6e0dc94c3341c358fca8eb2bf74e23aee3983ec4
 Pygments==2.0.2
-tinys3==0.1.11
+tinys3==0.1.12
 django-formtools==2.0
 django-two-factor-auth==1.6.0
 datadog==0.15.0
 django-websocket-redis==0.4.6
-django-redis-sessions==0.5.0
+django-redis-sessions==0.5.6
```

Comment:
https://github.com/martinrusev/django-redis-sessions/blob/master/CHANGELOG.md change log is new. no tags